### DISCUSSION TITLE
Startup Apps - often only about half is loaded - is Explorer Patcher responsible to load them and if so how to figure out the issue

### DISCUSSION THREAD

**Discussion by tmgeorge2** [valinet/ExplorerPatcher#3196] [Q&A]
Hi, starting W11 I should see 16 icons in the system tray. Quite often I only see about half of it. E.g. Steam isn't loaded.

Is there a way to analyze what's wrong?

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.
